add path to BamTools header files, otherwise some auxiliary programs don't compile
author: Albert Bogdanowicz
updated by: Ruben van Dijk (University of Groningen)
diff -Nru augustus.orig/auxprogs/bam2hints/Makefile augustus/auxprogs/bam2hints/Makefile
--- augustus.orig/auxprogs/bam2hints/Makefile	   2016-03-30 16:25:29.000000000 +0200
+++ augustus/auxprogs/bam2hints/Makefile	   2018-04-12 16:04:16.212990536 +0200
@@ -8,7 +8,7 @@
 #	Last modified:  09-October-2015 by Katharina J. Hoff
 
 # Variable definition
-INCLUDES = /usr/include/bamtools
+INCLUDES = $(EBROOTBAMTOOLS)/include/bamtools
 LIBS = -lbamtools -lz
 SOURCES = bam2hints.cc 
 OBJECTS = $(SOURCES:.cc=.o)
diff -Nru augustus.orig/auxprogs/filterBam/src/Makefile augustus/auxprogs/filterBam/src/Makefile
--- augustus.orig/auxprogs/filterBam/src/Makefile  2016-04-20 16:32:58.000000000 +0200
+++ augustus/auxprogs/filterBam/src/Makefile	   2018-04-12 16:04:43.033325771 +0200
@@ -9,7 +9,7 @@
 PROGRAM = filterBam
 OBJECTS = $(SOURCES:.cc=.o)
 BAMTOOLS = /usr/include/bamtools
-INCLUDES = -I$(BAMTOOLS) -Iheaders -I./bamtools
+INCLUDES = -I$(BAMTOOLS) -Iheaders -I./bamtools -I$(EBROOTBAMTOOLS)/include/bamtools
 LIBS = -lbamtools -lz
 CFLAGS = -std=c++0x
 VPATH = functions
--- augustus.orig/auxprogs/bam2wig/Makefile        2018-08-22 14:09:47.000000000 +0200
+++ augustus/auxprogs/bam2wig/Makefile             2018-08-22 14:10:36.000000000 +0200
@@ -15,13 +15,13 @@
 PROGRAM = bam2wig
 SOURCES = $(PROGRAM)
 OBJECTS = $(SOURCES:.c=.o)
-SAMTOOLS=$(TOOLDIR)/samtools/
-HTSLIB=$(TOOLDIR)/htslib/
-BCFTOOLS=$(TOOLDIR)/bcftools/
-TABIX=$(TOOLDIR)/tabix/
-INCLUDES=-I$(SAMTOOLS) -I. -I$(HTSLIB) -I$(BCFTOOLS) -I$(TABIX)
+SAMTOOLS=$(EBROOTSAMTOOLS)
+HTSLIB=$(EBROOTHTSLIB)
+BCFTOOLS=$(HTSLIB)
+TABIX=$(HTSLIB)
+INCLUDES=-I$(SAMTOOLS)/include/bam -I. -I$(HTSLIB)/include/htslib
 VPATH=$(SAMTOOLS)
-LIBS=$(SAMTOOLS)/libbam.a $(HTSLIB)/libhts.a -lcurses -lm -lz -lpthread
+LIBS=$(SAMTOOLS)/lib/libbam.a $(HTSLIB)/lib/libhts.a -lcurses -lm -lz -lpthread -lcurl -lssl -lcrypto
 CFLAGS=-Wall -O2 $(INCLUDES)
 CC=gcc
